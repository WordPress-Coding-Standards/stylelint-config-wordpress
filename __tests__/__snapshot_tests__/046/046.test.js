'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '046', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '046' )],
		configFile: systemTestUtils.caseConfig( '046' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
