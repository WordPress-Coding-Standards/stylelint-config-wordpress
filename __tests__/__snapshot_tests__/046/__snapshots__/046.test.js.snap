// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`046 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "046/stylesheet.css",
    "warnings": Array [
      Object {
        "column": 42,
        "line": 123,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 38,
        "line": 178,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
    ],
  },
]
`;
